lib/matplotlib/cbook/__init__.py

```diff
@@ -622,9 +622,6 @@ def __len__(self):
     def __getitem__(self, ind):
         return self._elements[ind]
 
-    def as_list(self):
```

Comment:
Why is this method removed?

Reply:
It was moved from `_AxesStack` to the base class; now that `_AxesStack` is back, it's not needed there. Better not to add more public API if not necessary.